frontend/src/components/Collections/components/Dataset/components/DownloadDataset/components/Content/index.tsx

```diff
@@ -43,6 +43,39 @@ const Content: FC<Props> = ({ onClose, name, dataAssets }) => {
       setFileSize,
       setIsLoading,
     });
+
+    async function getDownloadLink({
```

Comment:
moved this function inside the useEffect hook, since linter says the function wasn't in the dependency list `[format, dataAssets]`